riscv/csrs.cc

```diff
@@ -893,6 +896,7 @@ bool medeleg_csr_t::unlogged_write(const reg_t val) noexcept {
     | (1 << CAUSE_LOAD_PAGE_FAULT)
     | (1 << CAUSE_STORE_PAGE_FAULT)
     | (proc->extension_enabled('H') ? hypervisor_exceptions : 0)
+    | (proc->extension_enabled(EXT_ZICFILP) ? (1 << CAUSE_SOFTWARE_CHECK_FAULT) : 0)
```

Comment:
Please rename "SOFTWARE_CHECK_FAULT" to "SOFTWARE_CHECK"

Reply:
This name is generated by `riscv-opcodes`, and that particular line giving this name was contributed long ago (by you) and is not touched in this PR series. Given that `riscv-opcodes` generations are included in other projects, I think it's a bit too late to rename what's already in it.